bin/crew

```diff
@@ -1625,10 +1623,18 @@ def update_sha256(package, key, value)
 end
 
 def copy_package(pkg_name, prompt_msg = '')
+  pkg_file = File.join(CREW_LOCAL_REPO_ROOT, 'packages', "#{pkg_name}.rb")
+  # Use rubocop to sanitize package file, and let errors get flagged.
+  if @device[:installed_packages].any? { |elem| elem[:name] == 'ruby_rubocop' }
```

Comment:
I sure wish we had something like `if package_installed?('ruby_rubocop')`.  That would be more elegant.